provisioning_templates/PXEGrub2/kickstart_default_pxegrub2.erb

```diff
@@ -26,13 +33,13 @@ set default=0
 set timeout=<%= host_param('loader_timeout') || 10 %>
 
 menuentry '<%= template_name %>' {
-  <%= linuxcmd %> <%= @kernel %> ks=<%= foreman_url('provision') %> <%= pxe_kernel_options %> <%= snippet("kickstart_kernel_options").strip %> BOOTIF=01-$net_default_mac
```

Comment:
Is there a reason for removing the leading spaces?

Reply:
No my editor screwed it up.